app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,150 @@
+package com.codesoom.assignment;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpHandler;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.*;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static com.codesoom.assignment.HttpMethod.*;
+
+public class DemoHttpHandler implements HttpHandler {
+    private static final int HTTP_OK_CODE = 200;
+    private static final int HTTP_CREATE_CODE = 201;
+    private static final int HTTP_NO_CONTENT_CODE = 204;
+    private static final int HTTP_NOT_FOUND_CODE = 404;
+    private final TaskRepository repository;
+
+    public DemoHttpHandler(TaskRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException, IllegalArgumentException {
+        ObjectMapper mapper = new ObjectMapper();
+
+        HttpMethod method = httpMethod(exchange);
+        String path = httpRequestPath(exchange);
+        String body = httpRequestBody(exchange);
+
+        if (method == GET && path.equals("/tasks")) {
+            sendResponse(exchange, HTTP_OK_CODE, tasksToJson(mapper));
+        }
```

Comment:
```suggestion
        if (method == GET && path.equals("/tasks")) {
            sendResponse(exchange, HTTP_OK_CODE, tasksToJson(mapper));
            return
        }
```

할 일이 끝나면 빠르게 종료하여, 다음 코드를 읽을 필요도 없이 만들어서, 코드를 읽는 사람에게 부하를 줄이면 좋을 것 같아요.

See also
  - https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
동의합니다. 보호구문 개념을 적용하면 읽는 입장에서 더 편하겠네요~